libraries/core_libs/consensus/src/final_chain/rewards_stats.cpp

```diff
@@ -108,13 +108,6 @@ std::vector<addr_t> RewardsStats::processStats(const PeriodData& block, uint64_t
   txs_validators.reserve(block.transactions.size());
 
   for (const auto& tx : block.transactions) {
-    // TODO: if enabled, it would break current implementation of RewardsStats
-    // if (replay_protection_service_ && replay_protection_service_->is_nonce_stale(trx->getSender(),
-    //   trx->getNonce())) {
-    //   removeTransaction(tx.getHash());
```

Comment:
Please also remove commented removeTransaction method, as it was left only because there was link here to it